zebrad/tests/acceptance.rs

```diff
@@ -241,13 +255,17 @@ fn start_args() -> Result<()> {
     let output = child.wait_with_output()?;
     output.assert_failure()?;
 
+    // Check that we do not generate cache_dir
+    let cache_dir = tempdir.join("state");
+    assert!(!cache_dir.exists());
```

Comment:
We want to make sure that sled doesn't use the directory.

```suggestion
    // Check that we do not generate cache_dir
    let cache_dir = tempdir.join("state");
    assert!(!cache_dir.exists() || cache_dir.read_dir()?.count() == 0);
```

Reply:
Also, can we split this check out into a separate test, which just checks the `ephemeral` option?

See my earlier comment for some background:
https://github.com/ZcashFoundation/zebra/pull/939/files#r475980074